File: StudentSystem/ClientApp/src/app/models/kimlik.ts
import { Iletisim } from './Iletisim';
import { Kullanici } from './kullanici';
export class Kimlik {
  id: number;
  tcno: number;
  ad: string;
  soyad: string;
  dogumYeri: string;
  dogumTarihi: string;
  ilatisimId: number;
  iletisim: Iletisim;
  kullanici: Kullanici;
}

File: StudentSystem/Models/Kimlik.cs
using Microsoft.EntityFrameworkCore;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace StudentSystem.Model
{
    [Index(nameof(TCNO), IsUnique = true)]
    public class Kimlik
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        [Key]
        public int ID { get; set; }
        public long TCNO { get; set; }
        public string Ad { get; set; }
        public string Soyad { get; set; }
        public string DogumYeri { get; set; }
        public DateTime DogumTarihi { get; set; }
        [ForeignKey("Iletisim")]
        public int IlatisimID { get; set; }

        public Iletisim Iletisim { get; set; }
        
        public Kullanicilar Kullanicilar { get; set; }


    }
}
